CMakeLists.txt

```diff
@@ -249,3 +252,21 @@ if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
     set_property(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY VS_STARTUP_PROJECT ${STARTUPPROJECT})
   ENDIF()
 endif()
+
+
+#########################################################
+set(CPACK_PACKAGE_NAME "lib3mf")
+set(CPACK_PACKAGE_VENDOR "3MF Consortium")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "lib3mf - An implementation of the 3D Manufacturing Format file standard")
+set(CPACK_PACKAGE_VERSION "${LIB3MF_VERSION_MAJOR}.${LIB3MF_VERSION_MINOR}.${LIB3MF_VERSION_MICRO}")
```

Comment:
I suppose setting these variables create lib3mf.VERSION.dylib

![image](https://github.com/3MFConsortium/lib3mf/assets/145573667/838b3d02-2675-4330-9d2e-77709140d446)

Reply:
But the other binaries lib3mf.2.dylib/lib3mf.dylib sizes are 1kb where as lib3mf.2.3.1.0 has approx 2Mb in size. My doubt is whether the binaries with old names work ?